packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/customauthenticatoradminconsole.md

```diff
@@ -8,12 +8,12 @@
       * **Preferred** - User verification is optional.
       * **Required** - User verification is required.
    * **Authenticator logo:** Select the logo that users see on authentication screens.
-      * **Browse files** - Click this button and select an SVG file from the file selection dialog.
-      * **Use default logo** - Click the lock icon to use the default logo.
+      * **Browse files** - Click and select an SVG file from the file selection dialog.
+      * **Use default logo** - Click Lock to use the default logo.
```

Comment:
shouldn't this be **Lock** ? Or does it not say Lock? If it's just the icon, go back to "the lock icon"